src/modules/ganalytics/AnalyticsProvider.tsx

```diff
@@ -187,3 +241,63 @@ export function useAnalytics() {
   }
   return context;
 }
+
+const listNames = {
+  featuredListings: 'Featured listings',
+  currentListings: 'Current listings',
+};
+
+// static function to augment track event
+export function addListingToTrackCall(listing: Listing) {
```

Comment:
My opinion is to remove all the info from the event that we can pulled from the database. If we want to do an analysis of listings we are going to go to the db not google analytics. 

I do see potential for faceting events of how many click throughs for buy now vs auctions where current bid is 0.5 so mixed so staying open to possibilities.

Reply:
I'd like to see the event prep happening in the view leveraging track. Don't want to overload the analytics provider knowledge of every entity making use of it.

Reply:
I'd like to keep the additional listing event properties for now if you don't mind too much. I agree serious (money/sol) analytics should happen at database level, but I really want to  experiment with those extra properties in GA.

Also agree to move the helper function out of the analytics provider for now. It could make sense to add it back in if we started tracking listings from multiple views in the future though.